.rubocop.yml

```diff
@@ -11,6 +11,12 @@ Metrics/LineLength:
   Max: 120
   IgnoreCopDirectives: true
 
+Metrics/ParameterLists:
+  Max: 6
+
+Metrics/CyclomaticComplexity:
+  Max: 7
+
```

Comment:
Is this needed and if yes, what should be the maximum value?
@fedetaglia Any suggetions?

Reply:
by default it is 5, I have added 6 params in the method, so instead of using `  # rubocop:disable Metrics/ParameterLists
` added in .rubocop.yml

Reply:
fixed